hil/ext/switches/dell.py

```diff
@@ -94,7 +94,7 @@ def _sendline(self, line):
     @staticmethod
     def connect(switch):
         """connect to the switch, and log in."""
-        console = pexpect.spawn('telnet ' + switch.hostname)
+        console = pexpect.spawn('ssh ' + switch.username + '@' + switch.hostname)
```

Comment:
I would add a comment here saying that the dell switch still expects a username even after we specified the username in the spawn command.